Source/WebCore/platform/graphics/PlatformTimeRanges.h

```diff
@@ -109,7 +109,8 @@ class PlatformTimeRanges {
             return range.m_start >= m_end;
         }
     };
-    
+
```

Comment:
Are there extra spaces here? (I can't see them, but the -+ diff chunk here suggests the number of spaces might have changed). There should be no spaces.

Reply:
There were extra spaces that I removed.